crates/starknet-server/tests/test_dump_and_load.rs

```diff
@@ -217,4 +247,171 @@ mod dump_and_load_tests {
 
         remove_file(dump_file_name);
     }
+
+    #[tokio::test]
+    async fn dump_without_transaction() {
+        // dump on exit
+        let dump_file_name = "dump_without_transaction";
+        let devnet_dump = BackgroundDevnet::spawn_with_additional_args(&[
+            "--dump-path",
+            dump_file_name,
+            "--dump-on",
+            "exit",
+        ])
+        .await
+        .expect("Could not start Devnet");
+
+        #[cfg(windows)]
+        {
+            // To send SIGINT signal on windows, windows-kill is needed
+            let mut kill = Command::new("windows-kill")
+                .args(["-SIGINT", &devnet_dump.process.id().to_string()])
+                .spawn()
+                .unwrap();
+            kill.wait().unwrap();
+        }
+
+        #[cfg(unix)]
+        {
+            let mut kill = Command::new("kill")
+                .args(["-s", "SIGINT", &devnet_dump.process.id().to_string()])
+                .spawn()
+                .unwrap();
+            kill.wait().unwrap();
+        }
```

Comment:
Can't L264-L281 be extracted into a separate funcitonand used in all places requiring process killing? I think there is another place with this exact same code.

Reply:
Fixed in 04ca6f2e8a6b18388572e86e6adddf68f72a3f5b